packages/calico/extra/create-calico-docker-network.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python
+"""
+This script creates the calico docker network if not exist, and configures
```

Comment:
```suggestion
Creates the calico docker network if it does not exist, and configures
```

Reply:
Couldn't do a multi line suggestion, so reworded the beginning to keep the line length under 80 ;).